docs/deploy.md

```diff
@@ -1,108 +1,75 @@
 
 ### dependsOn
-`deploy` depends on `libertyStart` for a running server.
-
-### Properties
-
-See the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration) properties for common server configuration.
+`deploy` depends on any tasks of type `war` or `ear` so the package is ready before installation.  
+`deploy` also depends on `libertyCreate` to ensure that the server exists.
 
-The `deploy` task uses a `deploy` block inside the `liberty` block to define task specific behavior.
+### Parameters
 
-| Attribute | Type | Since | Description | Required |
-| --------- | ---- | ----- | ----------- | ---------|
-| file| String | 1.0 | Location of a single application to be deployed. The application type can be war, ear, rar, eba, zip, or jar. | Yes, only when a single file will be deployed. |
-| dir| String | 1.0 | Location of the directory where are the applications to be deployed.| Yes, only when multiples files will be deployed and `file` is not specified.|
-| include| String | 1.0 | Comma or space-separated list of patterns of files that must be included. All files are included when this attribute is omitted.| No |
-| exclude| String | 1.0 | Comma or space-separated list of patterns of files that must be excluded. No files are excluded when this attribute is omitted.| No |
+The `deploy` task can be configured using the `deploy` block. Below are the properties that can be set in the `deploy` block.
 
-### Examples
+| Attribute | Type  | Since | Description | Required |
+| --------- | ----- | ----- | ----------- | -------- |
+| apps | Object[] | 3.0 | Specifies which tasks/files should be deployed to the `apps` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
+| dropins | Object[] | 3.0 | Specifies which tasks/files should be deployed to the `dropins` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
 
-1. Deploys a single file.
-  ```groovy
-    apply plugin: 'liberty'
+In addition to the `deploy` block, the `stripVersion` and `looseApplication` properties are used for application installation. More information on these properties can be found in the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration).
 
-    liberty {
-        installDir = 'c:/wlp'
+Loose Application is only supported by Gradle 4.0 or higher for tasks of type ear.
 
-        server {
-            name = 'myServer'
+If you target an application to the `dropins` folder, no configuration is needed to get the server to start the application. When you want to configure an application in server.xml, the `location` is relative to the `apps` folder. Applications in the `apps` folder are not started without configuration. If you configure an application to install to the `apps` folder and it is not configured, default configuration is created in the `configDropins` folder.
```

Comment:
When targeting an application to the `dropins` folder, the application will start automatically when the server is running. No additional configuration is needed. When targeting an application to the `apps` folder, the application must be configured in the `server.xml` file.  Note that the `location` attribute is relative to the `apps` folder. If you configure an application to deploy to the `apps` folder and do not specify application configuration in the `server.xml` file, then default configuration is generated in the `configDropins` folder for the application.